README.md

```diff
@@ -100,10 +100,10 @@ Download the binary from Github Releases and put it in your `$PATH`.
 
 ### From the AUR
 
-Use your favorite AUR helper.
+Use your favorite AUR helper to install `trashy` or `trashy-git`. Trashy git will remain up to date with the `master` branch.
```

Comment:
I think it might be better to call out that the latest release version breaks tab completion, so new users don't go down a rabbit hole trying to fix it. Rather than just offering both versions with no context. At least until the release includes that fix.